www/docs/customization/chocolatey.md

```diff
@@ -0,0 +1,127 @@
+# Chocolatey Packages
+
+GoReleaser can also generate `nupkg` packages.
+[Chocolatey](http://chocolatey.org/) are packages based on nupkg format, that
+will let you publish your project directly to the Chocolatey Repository.From
+there it will be able to install in Windows distributions.
```

Comment:
maybe clarify that users will be able to install the app in their machines or something like that?

Reply:
Something like: 

> From there it will be able to install locally or in any Windows distributions.